tools/azure/cleanup.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env nash
+
+resgroups <= azure group list --json
+length <= echo $resgroups | jq ". | length"
+
+echo $length
```

Comment:
@tiago4orion I was able to get the length of the returned JSON array, how is the best way to iterate using this length so I can process each entry individually ? My plan is to ask for confirmation and delete one by one :-)